src/components/SwipeableItem.tsx

```diff
@@ -206,21 +221,29 @@ class SwipeRow extends React.Component<SwipeRowProps> {
                                     { scale: this.iconAnimState.scaling },
                                     { translateX: this.iconAnimState.position },
                                 ],
-                                borderColor: color(
-                                    255,
-                                    0,
-                                    0,
-                                    this.iconAnimState.colorVisible,
-                                ),
-                                borderWidth: this.iconAnimState.colorVisible,
-                                borderRadius: 20,
                             }}
                         >
```

Comment:
improvements made to the trash can icon animation. it now changes color to red when it reaches the threshold